app/controllers/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class RentsController < ApiController
+  def index
+    render json: Rent.all.page(params[:page])
+  end
+
+  def create
+    rent = Rent.new(rent_params)
+    if rent.save
+      render json: rent
+    elsif rent.book_id.nil? || rent.user_id.nil? || rent.to.nil? || rent.from.nil?
+      render json: { status: 'error',
```

Comment:
You're returning a success code here, even if the json contains the key code, you could do something like `render json: { errors: rent.errors.messages }` and set an error code to the response